pullrequests/PollPR.py

```diff
@@ -251,24 +282,53 @@ def PRBuilding(data, ErroCounter, g, org, counter, myfile, tbd_list, timetoken):
         ErroCounter = ErroCounter+1
 
     USER = data["user"]["login"]
-    user = g.get_user(USER)
+    UserOK=False
+    #global organization_users
 
-    if (org.has_in_members(user)):
-        print(
-            f"---> The user '{USER}' is a member of the organization '{ORGANIZATION}'.")
+    if (USER in organization_users):
+        UserOK=True
+        print(f"---> Earlier check found:'{USER}' is a member of the organization '{ORGANIZATION}'.")
+
+    else:
+        print('User was not checked previously for organization membership, checking now')
+        try:
+            user = g.get_user(USER)
+
+        except UnknownObjectException as e:
+            print(f"Did not get Github PR creator. Maybe Rate limit occured? Error: {e}")
+            ExitProg(4)
+        except Exception as e:
+            print(f"Did not get Github PR creator. General Syntax Error? Maybe Rate limit occured? Error: {e}")
+            ExitProg(4)
+
+        if (org.has_in_members(user)):
+            print(
+                f"---> The user '{USER}' is a member of the organization '{ORGANIZATION}'.")
+            organization_users.append(USER)
+            UserOK = True
+            # if (counter in tbd_list):
+            #    print(f"------> Handling PR number:{counter}")
+            #    if (ErroCounter == 0):
+
+            #        PRActions(SOURCE, counter, TARGET, myfile,
+            #              USER, SOURCE_REPO, timetoken)
+
+            #    else:
+            #        print("Errors in PR data from Github, not doing build activities")
```

Comment:
No out commented code to final commit, except maybe in cases where it acts as documentation.

Reply:
gone now